test/e2e/resources.cy.js

```diff
@@ -1,63 +1,74 @@
-import Course from '../../../src/course/en/course.json'
-
 describe('Resources', () => {
-    const checkDrawerLength = (count) => {
-        cy.get('.drawer__item').not('.u-display-none').should('have.length', count)
-      }
+  const ITEM_TYPES = [
+    'document',
+    'link',
+    'media'
+  ];
 
-    beforeEach(() => {
-        cy.visit('/');
-        cy.get('button[data-event="toggleDrawer"]').click()
-    })
+  const checkDrawerLength = (count) => {
+    cy.get('.drawer__item').not('.u-display-none').should('have.length', count);
+  };
 
-    it('should appear on the right hand side in menu view', () => {
-        cy.get('.drawer').should('have.css', 'right').and('match', /0px/)
+  const populateItemCounts = () => {
+    ITEM_TYPES.forEach(type => {
+      this.itemsCount[type] = this.resourceItems.filter(item => item?._type === type).count();
     });
+  };
 
-    it('should appear on the right hand side in course view', () => {
-        cy.get('button.drawer__close-btn').click()
-        cy.get('.menu-item .menu-item__button-container button').contains('View').first().click()
-        cy.get('button[data-event="toggleDrawer"]').click()
-        cy.get('.drawer').should('have.css', 'right').and('match', /0px/)
-    });
+  beforeEach(() => {
+    cy.getConfig();
+    this.resourceItems = this.config?._resources?._resourcesItems || [];
+    populateItemCounts()
 
-    it(`should show ${Course._resources._resourcesItems.length} items`, () => {
-        checkDrawerLength(4)
-    });
+    cy.visit('/');
+    cy.get('button[data-event="toggleDrawer"]').click();
+  });
 
-    it('should display the correct amount of items in each tab', () => {
-        cy.get('button.is-selected[id="resources__show-all"]').should('exist')
+  it('should appear on the right hand side in menu view', () => {
+    cy.get('.drawer').should('have.css', 'right').and('match', /0px/);
+  });
+
+  it('should appear on the right hand side in course view', () => {
+    cy.get('button.drawer__close-btn').click();
+    cy.get('.menu-item .menu-item__button-container button').contains('View').first().click();
+    cy.get('button[data-event="toggleDrawer"]').click();
+    cy.get('.drawer').should('have.css', 'right').and('match', /0px/);
```

Comment:
This should be rtl aware if you're intent on checking layout stuff. Should we do layout bits considering the themes might be wildly different?